Lib/test/test_asyncgen.py

```diff
@@ -399,11 +399,8 @@ async def gen():
 
         with self.assertWarns(DeprecationWarning):
             x = gen().athrow(GeneratorExit, GeneratorExit(), None)
-        with self.assertWarnsRegex(RuntimeWarning,
-                f"coroutine method 'athrow' of '{gen.__qualname__}' "
-                f"was never awaited"):
-            del x
-            gc_collect()
+        with self.assertRaises(GeneratorExit):
+            x.send(None)
```

Comment:
This test seems to pass even without the patch. What is the `x.send(None)` for here?

Reply:
This might also need a gc.collect()

Reply:
It's still passing on vanilla 3.12? What is the expected result without your patch?